src/esp/gfx/CubeMap.h

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree
+
+#ifndef ESP_GFX_CUBEMAP_H_
+#define ESP_GFX_CUBEMAP_H_
+
+#include <map>
+
+#include <Corrade/Containers/EnumSet.h>
+#include <Magnum/GL/CubeMapTexture.h>
+#include <Magnum/GL/Framebuffer.h>
+#include <Magnum/GL/Renderbuffer.h>
+#include <Magnum/Magnum.h>
+#include <Magnum/PixelFormat.h>
+#include <Magnum/ResourceManager.h>
+#include <Magnum/Trade/AbstractImporter.h>
+#include "esp/gfx/CubeMapCamera.h"
+#include "esp/gfx/RenderCamera.h"
+#include "esp/scene/SceneGraph.h"
+#include "esp/scene/SceneNode.h"
+
+namespace esp {
+namespace gfx {
+class CubeMap {
+ public:
+  static void enableSeamlessCubeMapTexture();
```

Comment:
Why not just call this implicitly on construction or something? It's always good to have this enabled, and this makes it too easy to forget (and the function doesn't seem to be used anywhere in this PR).

Reply:
I simply hope it can be called once, not every time in the constructor.

I can change.